webpack/containers/Application/config.js

```diff
@@ -11,11 +11,11 @@ export const links = [
   },
   {
     text: 'RH Subscriptions',
-    path: 'xui/subscriptions',
+    path: 'subscriptions',
```

Comment:
the subscriptions page renders with empty state, even when I have subscriptions on the old page:
![screenshot from 2018-05-22 16-42-25](https://user-images.githubusercontent.com/11807069/40386196-b4bf6f04-5e11-11e8-953e-f42310017d1e.png)


The problem here is probably because somehow katello uses react 16.2, as you can see here:


![screenshot from 2018-05-22 22-27-45](https://user-images.githubusercontent.com/11807069/40385924-e48df9cc-5e10-11e8-9fe4-a93124754ef2.png)

but we use `static getDerivedStateFromProps` in `SubscriptionTable.js` which is defined on 16.3. switching this to `componentWillReceiveProps` works well (although this lifecycle method is deprecated).

nitpick: could you place this spinner to the center of the page:
![screenshot from 2018-05-22 22-27-46](https://user-images.githubusercontent.com/11807069/40386353-2a9e0dac-5e12-11e8-9360-9ccce47b9eef.png)

Reply:
> The problem here is probably because somehow katello uses react 16.2, as you can see here:

See http://projects.theforeman.org/issues/23690 and http://projects.theforeman.org/issues/23691

> nitpick: would this spinner look better on the center of the page ?

@rohoover I will change this to whatever you think is best, just let me know.

Reply:
@waldenraines 
Thanks for bringing this to my attention.

Yes please center it. Here you can see some of the design guidelines (http://www.patternfly.org/pattern-library/communication/loading-state/#design). It doesn't seem to mention the space at the top, so if you double the current space at the top that would be good with me.

Reply:
> Yes please center it.

Since the spinner itself isn't really part of this work I created a new issue for it - http://projects.theforeman.org/issues/23922

Reply:
And PR: https://github.com/Katello/katello/pull/7451